tests/unit/test_util.py

```diff
@@ -151,10 +151,22 @@ def test_jit_test(self):
         second call assesses if a performance improvement has occurred given the
         JIT compilation.
         """
-        mock_function = MagicMock()
-        coreax.util.jit_test(mock_function)
-        num_calls = mock_function.call_count
-        self.assertEqual(num_calls, 2)
+
```

Comment:
Whilst editing this test, could you please fix the typo that is already there in the docstring - 'The function jit_test is used to asses the performance' We should replaced 'asses' with `assess`